packages/itwinui-css/src/alert/alert.scss

```diff
@@ -9,35 +9,35 @@
 
   --_iui-alert-border-color: var(--iui-color-border);
 
-  border-radius: var(--iui-border-radius-1);
+  border-radius: var(--_iui-alert-border-radius, var(--iui-border-radius-1));
```

Comment:
Can you explain the reason for this change?

Reply:
Using CSS vars versus overriding.  Decided it's unnecessary for the point of this PR and reverted.